sway/input/cursor.c

```diff
@@ -637,7 +637,8 @@ void cursor_send_pointer_motion(struct sway_cursor *cursor,
 	cursor->previous.y = cursor->cursor->y;
 	cursor->previous.node = node;
 
-	if (node && config->focus_follows_mouse) {
+	if (node && (config->focus_follows_mouse == FOLLOWS_YES ||
+		config->focus_follows_mouse == FOLLOWS_ALWAYS)) {
```

Comment:
Style: please add one more indentation level to this line, so we can easily tell it's the condition and not the body.